transactions/README.md

```diff
@@ -97,36 +107,69 @@ fileName:transaction.json
 }</code><span class="line-numbers-rows" aria-hidden="true"><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span></span></pre></div>
 </details>
 
-There are two types of transactions: authorisation transactions and clearing transactions. **Authorisation transactions** are processed when the customer is making the payment in-store or online. They trigger the `transaction.auth` webhook event that you can use to notify or reward the user in your application in real-time.
+The Fidel API currently supports three types of transactions: authorisation transactions, clearing transactions and refund transactions.
+
+**Authorisation transactions** are processed when a purchase registers on a linked card. For example, when a customer is making a payment in-store or online in real-time. When a customer makes a payment with a linked debit/credit card in an auth-enabled location, the `transaction.auth` webhook is also triggered and the transaction object sent to your specified URL in real-time.
```

Comment:
`is making a payment` -> `makes a payment` flows smoother imo

is the hyphen needed in `in real-time`? it's surely `real-time data` as an adjective, but when something happens `in real time` (or `in store`), i'm not so sure about that. can we double check this with Tom please?